week-1/interpret/percentage-change.js

```diff
@@ -11,10 +11,10 @@ console.log(`The percentage change is ${percentageChange}`);
 
 // Read the code and then answer the questions below
 
-// a) How many function calls are there in this file? Write down all the lines where a function call is made
+// a) How many function calls are there in this file? Write down all the lines where a function call is made - line 4,5,8,10
```

Comment:
You've identified 3 calls correctly. What's the function that's called on line 8?

Reply:
I am not sure. Is line 8 not a function call?